scanner/Makefile

```diff
@@ -125,6 +130,11 @@ db-image: copy-db-scripts copy-db-image-scripts copy-db-sigs
 	@echo "+ $@"
 	$(DOCKERBUILD) -t stackrox/$(image-prefix)-db:$(TAG) -f image/db/Dockerfile image/db
 
+.PHONY: clean-image
+clean-image:
```

Comment:
Warning: This might imply you are cleaning the local container images as well.

Reply:
i'll revert